### PR TITLE
Remove help text test which causes CI hangs

### PR DESCRIPTION
**Pull request by mfurtak**
The idea would be to revisit this test once we reach the next stage of our CI evolution. Right now it is unfortunately causing more trouble than it is worth.

### CODE REVIEW

**Comment by KrauseFx** [comment]
Looks good :+1:

**Comment by asfalcone** [comment]
:cactus: :cactus: :cactus:

File: fastlane/Fastfile [hunk 1]
```diff
@@ -119,13 +117,18 @@ private_lane :validate_repo do |options|
...
 
-  # Verifying the --help command
+  # Restricted from running on Travis CI because it causes builds to hang.
   #
-  binary_path = File.join("..", "bin", tool_name)
-  unless no_binary.include? tool_name
-    content = sh "PAGER=cat #{binary_path} --help"
-    ["--version", "https://fastlane.tools", tool_name].each do |current|
-      UI.user_error!("--help missing information: '#{current}'") unless content.include? current
+  unless ENV['TRAVIS']
```
Comment (KrauseFx) [inline fastlane/Fastfile]: This should be 
```ruby
FastlaneCore::Helper.ci?
```
instead

Comment (mfurtak) [inline fastlane/Fastfile]: Discussed offline, `FastlaneCore::Helper.ci?` would exclude all CIs, but we'd like to allow for testing on alternative CIs to see if they have the same hanging problem.

**Pull request merged**